CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Unreleased
+
+### Changed
+
+- Use `published` as release type for `release` workflow.
```

Comment:
This is **how** you did it. You should express **what** was done, instead. Also, it's a fix.

Anyway, you should ask yourself if it's something worth mentioning in the Changelog. Will users of the library be interested in such a change? You shouldn't even make a release for that.

Reply:
Probably users won't be interested. I removed it.